### DISCUSSION TITLE
part-ii/2016-48/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#271] [Question comments]
Part II 2016 Dynamical Systems: Paper 1, Section II, E 

https://questions.tripos.org/part-ii/2016-48/

**Comment by eha1234** [comment]
<details><summary>Second part</summary>

Use $p=1$ and note that trajectories can't cross $x=0$. Hamiltonian, 
use $V = -x^2/2 + x^3/3 = y^2x^2/2$
</details>

For stable manifold $y=-1+x/(3+a) + x^2/(2+a)(3+a)^2+...$ or something like this? Highly likely wrong.
